lib/computer.rb

```diff
@@ -0,0 +1,7 @@
+class Computer
+
+  def computermove
+  comp = ["Rock", "Paper", "Scissors"].sample
```

Comment:
There's a few places throughout where you could keep rubocop happier by indenting differently :D